templates/zerver/register.html

```diff
@@ -227,7 +227,7 @@ <h1>You&rsquo;re almost there.</h1>
                 </div>
                 {% endif %}
                 <div class="register-button-box">
-                    <button class="register-button" type="submit">{{ _('Sign up') }}</button>
+                    <button class="register-button" id="register-button" type="submit" disabled>{{ _('Sign up') }}</button>
```

Comment:
I think you give a more unique name for the ID instead of `register-button`. The `register-button` class name is not unique and is used in create_realm page as well. So using the same class name for the ID is not ideal.

Reply:
I could not get a better name for id, so I had made it `registration-button-id` for now